core/src/main/resources/reference.conf

```diff
@@ -128,6 +128,11 @@ dataRepoEntityProvider {
   bigQueryMaximumBytesBilled = 100GB
 }
 
+leonardo {
+    wdsType = "CROMWELL"
+    server = "https://leonardo.dsde-dev.broadinstitute.org"
```

Comment:
just wanted to check that hardcoding to "dev" is intentional?

Reply:
since this will get overridden by the values in https://github.com/broadinstitute/firecloud-develop/pull/3274, it would only cause a problem if the value in rawls.conf is missing and it falls back to using this value.

might be safest to use something like `server = "add-leo-hostname-here"` to catch that corner case?